react/fields/MonthPicker/NativeMonthPicker/NativeMonthPicker.less

```diff
@@ -16,13 +16,13 @@
   border-radius: @field-border-radius;
   color: @sk-charcoal;
   background-color: @sk-white;
+  appearance: none;
 
   &::-webkit-inner-spin-button,
   &::-webkit-calendar-picker-indicator,
   &::-webkit-clear-button {
     display: none;
     -webkit-appearance: none;
```

Comment:
this is redundant with `appearance: none` being outside this pseudo selector. confirming that it still works as expected if its on the class not the pseudo element?